tests/legacy-cli/e2e/tests/build/styles/tailwind-v2.ts

```diff
@@ -4,13 +4,8 @@ import { ng, silentExec } from '../../../utils/process';
 import { expectToFail } from '../../../utils/utils';
 
 export default async function () {
-  // Tailwind is not supported in Node.js 10
-  if (process.version.startsWith('v10')) {
```

Comment:
I'm assuming this is ok to remove because we don't support Node v10 in Angular CLI v13.1.x? Slightly unrelated: Have we explicitly documented our Node support anywhere? Best I could find was on our [getting started page](https://angular.io/guide/setup-local#prerequisites), but I'm not sure if that applies to all CLI versions, and I know we care about Node minor releases in some instances.

Reply:
Indeed that is the only place we list the node versions that we support. I think they could be better listed in table or something like that.